pkg/rpc/response/result/invoke.go

```diff
@@ -1,19 +1,28 @@
 package result
 
 import (
+	"encoding/hex"
 	"encoding/json"
+	"errors"
+	"fmt"
 
+	"github.com/nspcc-dev/neo-go/pkg/config/netmode"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
 	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
 )
 
 // Invoke represents code invocation result and is used by several RPC calls
-// that invoke functions, scripts and generic bytecode.
+// that invoke functions, scripts and generic bytecode. It's up to you to set
```

Comment:
... Transaction is represented in raw serialized format, use transaction.NewTransactionFromBytes or GetTransaction method to deserialize it.

Reply:
Added.